beacon.go

```diff
@@ -30,25 +31,40 @@ import (
 )
 
 type beaconConfig struct {
-	projectID string
-	tableId   string
+	ApiVersion string
+	ProjectID  string
+	TableId    string
 }
 
 const (
-	projectKey = "GOOGLE_CLOUD_PROJECT"
-	bqTableKey = "GOOGLE_BIGQUERY_TABLE"
+	apiVersionKey = "VERSION"
+	projectKey    = "GOOGLE_CLOUD_PROJECT"
+	bqTableKey    = "GOOGLE_BIGQUERY_TABLE"
 )
 
-var config = beaconConfig{
-	projectID: os.Getenv(projectKey),
-	tableId:   os.Getenv(bqTableKey),
-}
+var (
+	aboutTemplate = template.Must(template.ParseFiles("about.xml"))
+	config        = beaconConfig{
+		ApiVersion: os.Getenv(apiVersionKey),
+		ProjectID:  os.Getenv(projectKey),
+		TableId:    os.Getenv(bqTableKey),
+	}
+)
 
 func init() {
-	http.HandleFunc("/", handler)
+	http.HandleFunc("/", aboutBeacon)
+	http.HandleFunc("/query", query)
+}
+
+func aboutBeacon(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		return
```

Comment:
You should probably return an error here - like 405

Reply:
done